tests/llmobs/test_llmobs_span_writer.py

```diff
@@ -7,107 +7,149 @@
 from ddtrace.llmobs._writer import LLMObsSpanWriter
```

Comment:
Could we rename this file to `test_llmobs_span_agentless_writer.py` for clarity?